streaming-binlog/src/main/java/org/openmrs/analytics/model/DebeziumConfiguration.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package org.openmrs.analytics.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class DebeziumConfiguration {
```

Comment:
Is there a way to extend [debezium config](https://camel.apache.org/components/latest/debezium-mysql-component.html) It might be important to do this so that anyone can add new params in the json config without the need of modifying this model. As outlined [here](https://camel.apache.org/components/latest/debezium-mysql-component.html ) and [here](https://github.com/debezium/debezium/blob/b9028f42b1001c74ce136da4aa3be35c8250385a/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java#L870), we have over 80 parameters that can be configured. Would be nice to have the ability to do this without the need of recompiling debezium .  However, we might need to do this in another ticket, @bashir2  WDYT?

Reply:
i agree we implement this in another ticket since this will be out of scope for this ticket

Reply:
I think this is a good idea for a future ticket, please file an issue for that.